pkgs/applications/misc/warpd/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libXi
+, libXinerama
+, libXft
+, libXfixes
+, libXtst
+, libX11
+, libXext
+, waylandSupport ? false, cairo, libxkbcommon, wayland
+}:
+
+stdenv.mkDerivation rec {
+  pname = "warpd";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "rvaiya";
+    repo = "warpd";
+    rev = "v${version}";
+    sha256 = "yfqv+aABPSpw62BXpKXtBOp4CdmZfGSks+PojjsM/Vw=";
+    leaveDotGit = true;
+  };
+
```

You forgot about [this](https://github.com/rvaiya/warpd/blob/c3878e3bf69c97aa0f79f7bdf71d13cafbe1600b/Makefile#L1).
```suggestion
nativeBuildInputs = [ git ];
```
Or you can look in the repository on how to implement this with `postFetch`.